src/librustdoc/passes/collect_intra_doc_links.rs

```diff
@@ -859,26 +840,15 @@ impl<'a, 'tcx> LinkCollector<'a, 'tcx> {
         module_id: DefId,
         extra_fragment: &Option<String>,
     ) -> Option<Res> {
-        // resolve() can't be used for macro namespace
-        let result = match ns {
-            Namespace::MacroNS => self
-                .resolve_macro(path_str, item_id, module_id)
```

Comment:
Why doesn't this break? Isn't it missing macros, like I found in https://github.com/rust-lang/rust/pull/91427?

Reply:
If you look at `fn resolve_path`, which is the leaf path resolution function in rustdoc now, you can see that it falls back to `macro_rules` if nothing else is found. That part was done in https://github.com/rust-lang/rust/pull/95706.

Reply:
Added the test case.